src/lib/features/like-episode/model/like.ts

```diff
@@ -0,0 +1,51 @@
+import type { Episode } from '$lib/shared/api';
+import { persistentWritable, localStorageAdapter } from 'svelte-persistent-writable';
+
+/**
+ * The store containing set of episodes liked by the current user.
+ */
+export const likesStore = persistentWritable(new Map<Episode['id'], Episode>(), {
+  storage: localStorageAdapter('likes', {
+    serialize(likes: Map<Episode['id'], Episode>) {
+      const episodes: Episode[] = Array.from(likes.values());
+      return JSON.stringify(episodes);
+    },
+    deserialize(serialized: string) {
+      const episodes: Episode[] = JSON.parse(serialized);
+      return new Map(episodes.map((episode) => [episode.id, episode]));
+    },
+  }),
+});
+
+/**
+ * Register that the user liked the given episode.
```

Comment:
[praise](https://conventionalcomments.org/#labels): thank you for documenting your code, one does not see that often in the wild ;)